### PR TITLE
Update Tencent QQ icon

### PR DESCRIPTION
**Pull request by LitoMore**
**Popularity metric:** Similarweb global rank - `qq.com` - `#63`

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

According to their guideline, we should use the `QQ Blue` one instead.

The color was picked from https://qq.design/brand/BrandDesign/Color.

### CODE REVIEW

**Comment by uncenter** [comment]
(FYI you don't need to provide a popularity metric when updating an icon.)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15298,8 +15298,9 @@
         },
         {
             "title": "Tencent QQ",
-            "hex": "EB1923",
-            "source": "https://en.wikipedia.org/wiki/File:Tencent_QQ.svg"
+            "hex": "1EBAFC",
+            "source": "https://qq.design/brand/BrandDesign/Logo",
```
Comment (uncenter) [inline _data/simple-icons.json:15302]: Where are you getting a downloadable logo from? I'm only seeing PNG images.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json:15302]: I just modified the previous one.

Comment (uncenter) [reply _data/simple-icons.json:15302]: Ah. Well my point stands that the link listed isn't really a source then.

Comment (LitoMore) [reply _data/simple-icons.json]: OK. Then let's use the `https://en.wikipedia.org/wiki/File:Tencent_QQ.svg` instead.

Updated.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**